script.py

```diff
@@ -8,10 +8,12 @@
 import torch
 from scipy.spatial.distance import cosine
 import pickle
-from torchvision import transforms
-from torchvision import transforms, datasets
+from torchvision import (
+    transforms, 
+)
```

Comment:
if you are extracting transforms only, then `from torchvision import transforms`